src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/IO/TdsStream.cs

```diff
@@ -117,32 +111,29 @@ public override void SetLength(long value)
         /// <inheritdoc />
         public override void Write(ReadOnlySpan<byte> buffer)
         {
-            throw new NotImplementedException();
+            _writeStream.Write(buffer);
         }
 
         /// <inheritdoc />
         public override void Write(byte[] buffer, int offset, int count)
         {
-            throw new NotImplementedException();
+            _writeStream.Write(buffer, offset, count);
         }
 
         /// <inheritdoc />
-        public override ValueTask WriteAsync(
+        public override async ValueTask WriteAsync(
             ReadOnlyMemory<byte> buffer,
-            CancellationToken cancellationToken)
-        {
-            throw new NotImplementedException();
-        }
+            CancellationToken cancellationToken) 
```

Comment:
Resharper is unhappy because this should have a default value (ie, add ` = default`)

Reply:
@benrr101  for this, I want to force the callers to provide a value in this case, rather than using `=default` to make sure that the consumers are forced to think about passing down the Cancellation token.

Reply:
Resharper will also yell at you if you call a method with a cancellation token arg from a method that has a cancellation token arg 😄 

eg
```
public void MyAsync(CancellationToken ct) {
  something.TakesCancellationTokenAsync(); // <-- lights up "this method has an overload with cancellation support"
}
```

Not required change for PR - just sounds like we should use Resharper 😛